### ISSUE TITLE
里站画廊详情页解析失败 

### ISSUE THREAD

**Issue by Linnest2020** [exzhawk/EhViewer#35]
**简略描述 / Describe the bug**
大概说下是什么问题。
解析失败，画廊列表正常，画廊详情报错，里站应该也修改了
1.7.21也没有完全修复

**如何重现 / To Reproduce**
重现的步骤：
1. 切换至里站
2. 直接打开任意画廊详情
3. 出现报错页面

**预期行为 / Expected behavior**
应该是什么样子的？
和表站一样正常显示

**截图 / Screenshots**
能截图的话就发截图，请注意风序良俗。
NaN

**设备型号与 Android 版本 / Device model and Android version**
 - 设备型号：PHP110
 - Android 版本：14
 - 已经更新至 1.7.21

**备注 / Additional context**
还有什么想说的？
这个是里站的问题？

_Originally posted by @Linnest2020 in https://github.com/exzhawk/EhViewer/issues/32#issuecomment-2439850405_

**Comment by Linnest2020** [comment]
顺便因为表站是好的，因此和[之前的issue](https://github.com/exzhawk/EhViewer/issues/33#issue-2616112930)很不一样
那位暂时不清楚是里站还是表站
目前测试下来1.7.21表站有效。

**Comment by exzhawk** [comment]
Please provide your "Thumbnail Settings" in Settings.
I can't reproduce it with my account. please provide the preview **PART** of the HTML. not the full HTML in case of sensitive information.
something like this:

```html
        <div id="gdt" class="gt200">
            <a href="https://exhentai.org/s/96d06243c5/3102481-1">
                <div title="Page 1: 01.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/96/d0/96d06243c5ff903962b980be4a893149e80db3dd-448601-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
            </a>
            <a href="https://exhentai.org/s/bd962d6687/3102481-2">
                <div title="Page 2: 02.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/bd/96/bd962d66878d1215fe5441085e63998f287de489-317032-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
            </a>
            <a href="https://exhentai.org/s/64f4dacc95/3102481-3">
                <div title="Page 3: 03.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/64/f4/64f4dacc95f6a13a55eca635772bf0c8ce360252-437103-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
            </a>
            <a href="https://exhentai.org/s/1930398007/3102481-4">
                <div title="Page 4: 04.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/19/30/193039800725b7236e80fc1b8fa2cb51db83bf0b-342115-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
            </a>
            <a href="https://exhentai.org/s/8805cfb0ba/3102481-5">
                <div title="Page 5: 05.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/88/05/8805cfb0ba0ee48a448b6986eede8706d82cd812-425874-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
            </a>
            <a href="https://exhentai.org/s/0849fe6d2f/3102481-6">
                <div title="Page 6: 06.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/08/49/0849fe6d2f5f2d6fdc3fcaac187ef16b79db3583-290782-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
            </a>
        </div>

```

**Comment by Linnest2020** [comment]
> Please provide your "Thumbnail Settings" in Settings. I can't reproduce it with my account. please provide the preview **PART** of the HTML. not the full HTML in case of sensitive information. something like this:
> 
> ```
>         <div id="gdt" class="gt200">
>             <a href="https://exhentai.org/s/96d06243c5/3102481-1">
>                 <div title="Page 1: 01.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/96/d0/96d06243c5ff903962b980be4a893149e80db3dd-448601-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
>             </a>
>             <a href="https://exhentai.org/s/bd962d6687/3102481-2">
>                 <div title="Page 2: 02.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/bd/96/bd962d66878d1215fe5441085e63998f287de489-317032-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
>             </a>
>             <a href="https://exhentai.org/s/64f4dacc95/3102481-3">
>                 <div title="Page 3: 03.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/64/f4/64f4dacc95f6a13a55eca635772bf0c8ce360252-437103-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
>             </a>
>             <a href="https://exhentai.org/s/1930398007/3102481-4">
>                 <div title="Page 4: 04.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/19/30/193039800725b7236e80fc1b8fa2cb51db83bf0b-342115-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
>             </a>
>             <a href="https://exhentai.org/s/8805cfb0ba/3102481-5">
>                 <div title="Page 5: 05.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/88/05/8805cfb0ba0ee48a448b6986eede8706d82cd812-425874-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
>             </a>
>             <a href="https://exhentai.org/s/0849fe6d2f/3102481-6">
>                 <div title="Page 6: 06.jpg" style="width:200px;height:283px;background:transparent url(https://s.exhentai.org/t/08/49/0849fe6d2f5f2d6fdc3fcaac187ef16b79db3583-290782-1447-2048-jpg_l.jpg) 0 0 no-repeat"></div>
>             </a>
>         </div>
> ```

```
<div id="gdt" class="gt200">
  <a href="https://exhentai.org/s/76ba65bed0/3090756-1">
    <div>
      <div title="Page 1: 1.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/76/ba/76ba65bed0b73ce1a35fc3e819db2563792a0853-2910654-2150-3035-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 1</div></div>
  </a>
  <a href="https://exhentai.org/s/3e3a5b2aa8/3090756-2">
    <div>
      <div title="Page 2: 2.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/3e/3a/3e3a5b2aa80431927f9854677e3f2b2aa30a54f6-3372335-2150-3035-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 2</div></div>
  </a>
  <a href="https://exhentai.org/s/f4c91580ba/3090756-3">
    <div>
      <div title="Page 3: 3.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/f4/c9/f4c91580baa6ebffa810c03295f1ffa240258000-2561467-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 3</div></div>
  </a>
  <a href="https://exhentai.org/s/04eeb50040/3090756-4">
    <div>
      <div title="Page 4: 4.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/04/ee/04eeb5004050fb7f6feee6b4ed41100cafb2896f-2962586-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 4</div></div>
  </a>
  <a href="https://exhentai.org/s/a3e0f7bde8/3090756-5">
    <div>
      <div title="Page 5: 5.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/a3/e0/a3e0f7bde833910824702a52ecd158b1fd2e472a-3480474-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 5</div></div>
  </a>
  <a href="https://exhentai.org/s/fb700224f7/3090756-6">
    <div>
      <div title="Page 6: 6.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/fb/70/fb700224f715fd0bd56d7c65861da94e2e78515f-3055252-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 6</div></div>
  </a>
  <a href="https://exhentai.org/s/2a7d8d8387/3090756-7">
    <div>
      <div title="Page 7: 7.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/2a/7d/2a7d8d838774b8e4ffc20872136b413c94eaed89-3091824-2150-3035-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 7</div></div>
  </a>
  <a href="https://exhentai.org/s/768f3bd9b9/3090756-8">
    <div>
      <div title="Page 8: 8.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/76/8f/768f3bd9b90f98c0df6abb8f791b861072c40cec-3190264-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 8</div></div>
  </a>
</div>
```
Thumbnail Settings:
![7D@_QAGZJ8P{I$GPGF_%JQ_tmb](https://github.com/user-attachments/assets/50995ba7-a0f1-4950-a30a-755ee27555a1)

**Comment by Linnest2020** [comment]
Plus: Opening `Fix thumb url` still not works
![HWNY31DG89$S(3BQGMHCTIU_tmb](https://github.com/user-attachments/assets/4680cfac-c6dd-416b-b918-af3f7be65090)
![{XZ73X _N0~0FFSWW NSLFP_tmb](https://github.com/user-attachments/assets/5de25295-cb64-4fd8-8e91-ed33cb8d2e50)

**Comment by Linnest2020** [comment]
OK it is Ehwebsite Settings...
When `Gallery Page Thumbnail Labeling` is `None`, it works on 1.7.21
![P%HBL0GQ{O$~3)F6FE)O)1N](https://github.com/user-attachments/assets/083fb7bf-a4cd-4b38-a8cb-37e30979daf4)
But if  `Gallery Page Thumbnail Labeling` is `Page Number Only` by default, it can not run and raise Error
![Z0%TMT91OA2)YLPN {UT2 2](https://github.com/user-attachments/assets/7c62e2fd-ac61-4108-84d7-b17cbeeaaa13)

Further I provide the HTML parts in `Page Number + Name` for Enhancement(if it would enhance):
```
<div id="gdt" class="gt200">
  <a href="https://exhentai.org/s/76ba65bed0/3090756-1">
    <div>
      <div title="Page 1: 1.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/76/ba/76ba65bed0b73ce1a35fc3e819db2563792a0853-2910654-2150-3035-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 1: 1.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/3e3a5b2aa8/3090756-2">
    <div>
      <div title="Page 2: 2.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/3e/3a/3e3a5b2aa80431927f9854677e3f2b2aa30a54f6-3372335-2150-3035-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 2: 2.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/f4c91580ba/3090756-3">
    <div>
      <div title="Page 3: 3.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/f4/c9/f4c91580baa6ebffa810c03295f1ffa240258000-2561467-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 3: 3.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/04eeb50040/3090756-4">
    <div>
      <div title="Page 4: 4.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/04/ee/04eeb5004050fb7f6feee6b4ed41100cafb2896f-2962586-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 4: 4.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/a3e0f7bde8/3090756-5">
    <div>
      <div title="Page 5: 5.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/a3/e0/a3e0f7bde833910824702a52ecd158b1fd2e472a-3480474-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 5: 5.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/fb700224f7/3090756-6">
    <div>
      <div title="Page 6: 6.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/fb/70/fb700224f715fd0bd56d7c65861da94e2e78515f-3055252-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 6: 6.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/2a7d8d8387/3090756-7">
    <div>
      <div title="Page 7: 7.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/2a/7d/2a7d8d838774b8e4ffc20872136b413c94eaed89-3091824-2150-3035-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 7: 7.jpg</div></div>
  </a>
  <a href="https://exhentai.org/s/768f3bd9b9/3090756-8">
    <div>
      <div title="Page 8: 8.jpg" style="width:200px;height:282px;background:transparent url(https://s.exhentai.org/t/76/8f/768f3bd9b90f98c0df6abb8f791b861072c40cec-3190264-2090-2951-jpg_l.jpg) 0 0 no-repeat"></div>
      <div>Page 8: 8.jpg</div></div>
  </a>
</div>
```

**Comment by exzhawk** [comment]
fixed in 1.7.22

**Comment by exzhawk** [comment]
fixed in 1.7.22
